meson.build

```diff
@@ -38,7 +38,25 @@ endif
 
 install_subdir(
   'test',
-  install_dir: join_paths(get_option('datadir'), meson.project_name())
+  install_dir: join_paths(get_option('datadir'), meson.project_name()),
+  exclude_files: [
+    'system/libs/bats-assert/.git',
```

Comment:
Is it necessary to list `.git` here when it's listed in `exclude_directories` below?